collect_app/src/main/res/values/strings.xml

```diff
@@ -576,4 +576,5 @@
     <string name="newer_version_of_a_form_info">This is an updated version of a form you already have!</string>
     <string name="svg_file_does_not_exist">SVG file does not exist!</string>
     <string name="svg_not_supported_device">ImageMap widget requires Android 5 (Lollipop) or newer. Your device is not supported!</string>
+    <string name="no_image_chooser"> No suitable application to handle this request</string>
```

Comment:
Please remove that space at the beginning and I would add something more. Users should know that it's about camera apps.

Reply:
@grzesiek2010 could you please suggest something?

Reply:
The same we use in all other similar cases. Here is an example https://github.com/opendatakit/collect/blob/master/collect_app/src/main/java/org/odk/collect/android/widgets/ImageWidget.java#L194 instead of `image capture` you can put `QR code scan`

Reply:
@grzesiek2010 this PR addresses the issue of not having an image chooser application (like, gallery).
I have modified the string. Please have a look.

Reply:
ahh I missed it so I think you can use exactly this one https://github.com/opendatakit/collect/blob/master/collect_app/src/main/java/org/odk/collect/android/widgets/ImageWidget.java#L213